config/sudoers

```diff
@@ -32,17 +32,11 @@ vx-admin ALL=(root:ALL) NOPASSWD: /vx/code/app-scripts/unmount-usb.sh
 vx-admin ALL=(root:ALL) NOPASSWD: /vx/code/vxsuite/libs/auth/src/intermediate-scripts/create-cert
 vx-services ALL=(root:ALL) NOPASSWD: /vx/code/vxsuite/libs/auth/src/intermediate-scripts/create-cert
 vx-services ALL=(root:ALL) NOPASSWD: /vx/code/vxsuite/libs/auth/src/intermediate-scripts/sign-message
-vx-services ALL=(root:ALL) NOPASSWD: /vx/code/vxsuite/libs/usb-drive/src/mount.sh
-vx-services ALL=(root:ALL) NOPASSWD: /vx/code/vxsuite/libs/usb-drive/src/unmount.sh
+vx-services ALL=(root:ALL) NOPASSWD: /vx/code/vxsuite/libs/usb-drive/scripts/*
 vx-ui ALL=(root:ALL) NOPASSWD: /vx/code/config/admin-functions/timedatectl
 vx-ui ALL=(root:ALL) NOPASSWD: /usr/bin/brightnessctl
 vx-ui ALL=(root:ALL) NOPASSWD: /usr/local/bin/tpm2-totp
 vx-ui ALL=(root:ALL) NOPASSWD: /vx/code/app-scripts/sign.sh
 vx-ui ALL=(root:ALL) NOPASSWD: /vx/code/app-scripts/mount-usb.sh
 vx-ui ALL=(root:ALL) NOPASSWD: /vx/code/app-scripts/unmount-usb.sh
-vx-ui ALL=(root:ALL) NOPASSWD: /vx/code/vxsuite/libs/usb-drive/src/mount.sh
-vx-ui ALL=(root:ALL) NOPASSWD: /vx/code/vxsuite/libs/usb-drive/src/unmount.sh
```

Comment:
I don't think `vx-ui` needs these perms?

Reply:
@jonahkagan is there any particular reason this was added in?

Reply:
I think I was just trying to mirror the existing permissions I saw in the file